README.md

```diff
@@ -259,6 +258,74 @@ curl http://aws-java-sample-java-function.default.dev.triggermesh.io -d '{"event
 {"message":"Hello, the current time is Tue Apr 07 13:59:17 GMT 2020"}
 ```
 
+### Run in Docker 
+
+For cases in which the use of additional components (tm CLI, Tekton, Knative, k8s) is undesirable, it is possible to build KLR function as a standalone Docker container and run it in any environment. To do this, you should extract Dockerfile from the runtime you are interested in, put it in the directory with your function, update the handler variable, and build the container. Here is Dockerfile definition for all runtimes:
+
+- [go](https://github.com/triggermesh/knative-lambda-runtime/blob/9a74ce1ac03d56d233cfc7a46d84f2c5e5f2685a/go-1.x/runtime.yaml#L44-L68)
+- [java](https://github.com/triggermesh/knative-lambda-runtime/blob/9a74ce1ac03d56d233cfc7a46d84f2c5e5f2685a/java8/runtime.yaml#L43-L53)
+- [node-10](https://github.com/triggermesh/knative-lambda-runtime/blob/9a74ce1ac03d56d233cfc7a46d84f2c5e5f2685a/node-10.x/runtime.yaml#L43-L48)
+- [node-4](https://github.com/triggermesh/knative-lambda-runtime/blob/9a74ce1ac03d56d233cfc7a46d84f2c5e5f2685a/node-4.x/runtime.yaml#L43-L48)
+- [python-2](https://github.com/triggermesh/knative-lambda-runtime/blob/9a74ce1ac03d56d233cfc7a46d84f2c5e5f2685a/python-2.7/runtime.yaml#L43-L50)
+- [python-3](https://github.com/triggermesh/knative-lambda-runtime/blob/9a74ce1ac03d56d233cfc7a46d84f2c5e5f2685a/python-3.7/runtime.yaml#L43-L50)
+- [ruby-2](https://github.com/triggermesh/knative-lambda-runtime/blob/9a74ce1ac03d56d233cfc7a46d84f2c5e5f2685a/ruby-2.5/runtime.yaml#L43-L47)
+
+
+Let's build Python-3.7 function as an example:
+
+1. Create directory and save your function code there:
+
+```
+  mkdir python
+  cd python
+  cat > handler.py <<EOF
+import json
+import datetime
+def endpoint(event, context):
+  current_time = datetime.datetime.now().time()
+  body = {
+      "message": "Hello, the current time is " + str(current_time)
+  }
+  response = {
+      "statusCode": 200,
+      "body": json.dumps(body)
+  }
+  return response
+EOF
+```
+
+2. Extract runtime [Dockerfile](https://github.com/triggermesh/knative-lambda-runtime/blob/9a74ce1ac03d56d233cfc7a46d84f2c5e5f2685a/python-3.7/runtime.yaml#L43-L50), update `_HANDLER` variable and store it in the same directory:
+
+```
+  cat > Dockerfile <<EOF
+  FROM gcr.io/triggermesh/knative-lambda-python37
+  ENV _HANDLER handler.endpoint
+  COPY . .
+  RUN if [ -f requirements.txt ]; then pip3.7 install -r requirements.txt ;fi
+  ENTRYPOINT ["/opt/aws-custom-runtime"]
+EOF
+```
+
+`_HANDLER` variable in most cases consist of the filename without the file format part and function name.
```

Comment:
most cases ? what are the other cases ?

Reply:
It depends on the language and Lambda's implementation of the runtime. Here are some examples of uncommon cases:

- https://docs.aws.amazon.com/lambda/latest/dg/golang-handler.html
Golang: there is always must be the `main` package with the `main()` function, so in fact, Go runtime doesn't need a custom handler at all.
  
- https://docs.aws.amazon.com/lambda/latest/dg/java-handler.html
Java: Afaik, Java compiler puts everything in its paths and which only Java dev can figure out, i.e. handler may look like `com.serverless.Handler` and it will run code in `src/main/java/com/serverless/Handler.java` file. 

I really think that if someone wants to run his AWS Lambda functions in KLR he should be familiar with the "handler" thing. Otherwise, we should copy a big part of AWS documentation.